consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/RoundChangeCertificate.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.consensus.qbft.payload;
+
+import org.hyperledger.besu.consensus.common.bft.payload.SignedData;
+
+import java.util.Collection;
+import java.util.StringJoiner;
+
+public class RoundChangeCertificate {
```

Comment:
Didn't think the RoundChange certificate was needed for QBFT?

Reply:
it makes the creation of messages somewhat simplier, as it collates the various data required to construct a proposal message.

Reply:
Think a struct makes sense, but calling it a RoundChangeCertificate is a bit confusing as that is for a different purpose or at least that's what I remember. RoundChangePayMetadata?